src/service/chat.js

```diff
@@ -114,15 +114,15 @@ const sendAnonymousDMOtherProfile = async ({
   message
 }) => {
   const payload = {
-    members: [user_id],
+    member: user_id,
     message,
     anon_user_info_emoji_name,
     anon_user_info_emoji_code,
     anon_user_info_color_name,
     anon_user_info_color_code
   };
 
-  const response = await anonymousApi.post('/chat/init-chat-anonymous', payload);
+  const response = await anonymousApi.post('/chat/init-chat-anonymous-v2', payload);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The API endpoint has been changed from '/chat/init-chat-anonymous' to '/chat/init-chat-anonymous-v2'. Ensure that the new endpoint is correctly implemented and tested on the server-side before making this change.

```diff
-  const response = await anonymousApi.post('/chat/init-chat-anonymous', payload);
+  const response = await anonymousApi.post('/chat/init-chat-anonymous-v2', payload);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->